### ISSUE TITLE
Should schema2asp not be writing extent(F,MIN,MAX)?

### ISSUE THREAD

**Issue by trubens71** [uwdata/draco#93]
I note that schema2asp.ts is writing only field type and cardinality to the lp. Should it not also write MIN and MAX so that the 4 hard rules and 2 soft rules defined by "extent" come into play? I assume if it were to do so then the MIN/MAX values must be integers.

Same question for entropy but I think more complex as not in datalib summary stats.

**Comment by domoritz** [comment]
Yes, that would be better. Draco doesn't need the extent but it's useful to have. 

Could you send a pull request to add this feature?
